include/dali/c_api.h

```diff
@@ -343,7 +343,8 @@ daliOutputCopy(daliPipelineHandle *pipe_handle, void *dst, int output_idx, devic
  * @brief Copy the samples in output stored at position `output_idx` in the pipeline
  *        to scattered memory locations.
  * @param pipe_handle Pointer to pipeline handle
- * @param dsts Pointers to the destination buffers where each sample will be copied
+ * @param dsts Pointers to the destination buffers where each sample will be copied.
+ *             A nullptr dst pointer for a given sample will result in that sample not to be copied
```

Comment:
nitpick (a bit easier to understand?):
```suggestion
 *             A `nullptr` dst pointer for a sample will discard that sample.
```